react-components/intro-to-react-js.md

```diff
@@ -0,0 +1,101 @@
+# Intro to React JS
+
+## Learning Goals
+
+- What is React?
+- Why do we learn React?
+- Who is using React?
+- Intro to the Virtual DOM
+- Intro to Components
+
+## Introduction
+
+Consider a social media website, such as [Reddit.com](https://www.reddit.com/r/reactjs/). Every given web page needs to do soooo many things! To name a few, each web page needs to:
+
+- List posts, the number of upvotes, and the comments
```

Comment:
at least in the screen shot, upvotes and comments both seem to be numbers, and both are related to a post, so maybe the following phrasing to reflect that?

if this seems like an unnatural interpretation to a more native redittor, feel free to ignore!

```suggestion
- List posts, including the numbers of upvotes and comments
```